src/spectrecoff/Output.fs

```diff
@@ -135,51 +136,70 @@ let rec toMarkedUpString (payload: OutputPayload) =
     | MarkupCD (color, decorations, content) -> content |> markupString (Some color) decorations
     | MarkupC (color, content) -> content |> markupString (Some color) []
     | MarkupD (decorations, content) -> content |> markupString None decorations
-    | Link link -> link |> markupLink "" 
+    | Link link -> link |> markupLink ""
     | LinkWithLabel (label, link) -> label |> markupLink link //$"link={link}"
     | Emoji emoji -> emoji |> padEmoji
-    | NewLine -> Environment.NewLine
-    | Collection items ->
-        items
-        |> List.map (fun item ->
-            match item with
-            | Renderable _ -> failwith "Renderables can't be used in collections."
-            | BulletItems _ -> failwith "Bullet items can't be used within bullet items."
-            | Collection items ->
-                items
-                |> List.map toMarkedUpString
-                |> joinSeparatedBy " "
-            | _ -> toMarkedUpString item)
-        |> joinSeparatedBy " "
+    | NextLine -> ""
+    | EmptyLine -> " "
     | BulletItems items ->
         items
         |> List.map (fun item ->
             match item with
             | Renderable _ -> failwith "Renderables can't be used within bullet items."
             | BulletItems _ -> failwith "Bullet items can't be used within bullet items."
-            | Collection items -> CO ([C bulletItemPrefix]@items)
-            | _ -> CO [C bulletItemPrefix; item])
+            | _ -> Many [C bulletItemPrefix; item])
         |> List.map toMarkedUpString
         |> joinSeparatedByNewline
     | Many payloads ->
         payloads
         |> List.map toMarkedUpString
-        |> joinSeparatedByNewline
+        |> joinSeparatedBy " "
     | Renderable _ -> failwith "The payload type 'Renderable' is not stringifyable."
 
+let isStringifyable payload =
+    match payload with
+    | Vanilla _
+    | Calm _
+    | Pumped _
+    | Edgy _
+    | Emoji _
+    | Link _
+    | LinkWithLabel _
+    | MarkupL _
+    | MarkupC _
+    | MarkupD _
+    | MarkupCD _ -> true
+    | _ -> false
+
+let combineStringifyables item1 item2 =
+    Vanilla (item1 + " " + item2)
```

Comment:
What about `Vanilla ([item1; item2] |> joinSeparatedBy " ")`

Then, maybe even 

```
let combineStringifyables items =
    Vanilla (items |>  |> joinSeparatedBy " ")
```

and with that, teh calling site becomes clearer:

`reduceRenderables( [head1; head2] |> List.map toMarkedUpString |> combineStringifyables :: tail2)`

Reply:
yeah that is much better. not only does the call look neater, but it now also supports an arbitrary number of payloads instead of only 2. 
Maybe this will come in handy down the line :)

thanks